src/sections/header.html

```diff
@@ -1,3 +1,23 @@
-<header>
-    <h1 class="text-green-200">This the header</h1>
-</header>
\ No newline at end of file
+<header class="font-outfit lg:py-8  sm:py-8">
+    <nav class="fixed md:right-10 md:left-10 right-5 left-5 top-4 z-50">
+        <div class="flex justify-between ">
+            <div class="">
+                <a href="#"><p class="md:text-xl text-2xl">DE</p></a>
+                <a href="#"><p class="md:text-xl text-2xl">EN</p></a>
+            </div>
+            <h1 class="md:text-5xl md:font-extrabold flex items-center text-4xl font-semibold relative">Baumeister</h1>
+            <img src="src/assets/images/white-shape.svg" alt="white-shape" class="md:hidden block absolute -z-10 left-16 w-[13.5rem]">
+            <ul class="hidden lg:block lg:text-2xl">
+                <li><a href="#about">About</a></li>
+                <li><a href="#services">Services</a></li>
+                <li><a href="#reference">References</a></li>
+                <li><a href="../../contact.html">Contact</a></li>
+            </ul>
+            
+            <div class="lg:hidden flex items-center  w-[3rem] md:w-[5rem]" id="menu">
```

Comment:
same case

Reply:
Removed the arbitrary values and configured them in tailwind config file.